src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1264,6 +1281,32 @@ export class AmpStoryPlayer {
     );
   }
 
+  /**
+   * Rewinds the given story.
+   * @param {string} storyUrl
+   */
+  rewind(storyUrl) {
+    const storyIdx = storyUrl
+      ? findIndex(this.stories_, ({href}) => href === storyUrl)
+      : this.currentIdx_;
+
+    const story = this.stories_[storyIdx];
+    if (!story) {
+      throw new Error(`Story URL not found in the player: ${storyUrl}`);
+    }
```

Comment:
Let's not copy/paste this code into each single method we add, and create a method for it